src/core/meson.build

```diff
@@ -132,6 +132,8 @@ libcore_sources = '''
 subdir('bpf/socket_bind')
 if conf.get('BPF_FRAMEWORK') == 1
         libcore_sources += [socket_bind_skel_h]
+        subdir('bpf/restrict_fs')
```

Comment:
this commit "bpf: add meson build rules for the restrict_fs program" can be squashed in the previous one "bpf: add restrict_fs BPF program", it doesn't make much sense standalone?